crates/mir/src/emit/op_code.rs

```diff
@@ -44,4 +49,5 @@ pub enum MirInstructionData<'content> {
 #[derive(Debug, Default, PartialEq)]
 pub struct MirInstructionMeta {
     pub span: Option<Range<usize>>,
+    pub _default: (),
```

Comment:
What's this field supposed to do?

Reply:
For simpler type construction after adding new fields:
```rs
MirInstructionMeta {
    span: ...,
    ...Default::default()
}
```
This allow to add new fields without changing old code in parts when this change not affects it.

Reply:
Rust has an attribute for this: [`#[non_exhaustive]`](https://doc.rust-lang.org/reference/attributes/type_system.html#the-non_exhaustive-attribute)